opm/simulators/flow/LogOutputHelper.cpp

```diff
@@ -617,30 +718,32 @@ outputInjectionReportRecord_(const std::vector<Scalar>& wellInj,
         wellInj[WellProdDataType::WellLocationi] >= 1;
 
     std::ostringstream ss;
-
-    ss << std::right << std::fixed << std::setprecision(0) << ':'
-       << std::setw(8) << wellInjNames[WellInjDataType::WellName] << ':';
+    ss << fmt::format(":{:<8}:", wellInjNames[WellInjDataType::WellName]);
 
     if (! isWellRecord) {
-        ss << std::setw(11) << "" << ':';
+        ss << fmt::format("{:11}:", "");
     } else {
-        ss << std::setw( 5) << wellInj[WellInjDataType::WellLocationi] << ','
-           << std::setw( 5) << wellInj[WellInjDataType::WellLocationj] << ':';
+        ss << fmt::format("{:>3},{:>3} {:>3}:",
+                          wellInj[WellInjDataType::WellLocationi],
+                          wellInj[WellInjDataType::WellLocationj],
+                          "");
```

Comment:
So if we need three digits for both I and J, then we'll get output of the form
```
123,456
```
Is that intentional?

Reply:
yes, that is the intent. it is dictated by column width (assuming i'm gonna stay "compatible").